<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <widget class="QTabWidget" name="InputWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="EncryptTab">
         <attribute name="title">
          <string>Зашифровать</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="eKeyLabel">
              <property name="text">
               <string>Ключ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="eKeyEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="eTextLabel">
              <property name="text">
               <string>Открытый текст</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="eTextEdit"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="encryptButton">
                <property name="text">
                 <string>Зашифровать</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="DecryptTab">
         <attribute name="title">
          <string>Расшифровать</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="dKeyLabel">
              <property name="text">
               <string>Ключ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="dKeyEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="dTextLabel">
              <property name="text">
               <string>Шифротекст</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="dTextModeEdit">
              <property name="editable">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>Raw (сырые байты в HEX)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>UTF-8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ASCII</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="dTextEdit"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="decryptButton">
                <property name="text">
                 <string>Зашифровать</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="OutputWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="RawDataTab">
         <attribute name="title">
          <string>Raw</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="RawTextBrowser"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="UtfTab">
         <attribute name="title">
          <string>UTF-8</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="UtfTextBrowser"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="AsciiTab">
         <attribute name="title">
          <string>ASCII</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="AsciiTextBrowser"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
